tests/foreman/api/test_errata.py

```diff
@@ -399,21 +410,18 @@ def package_applicability_changed_as_expected(
         )
     if return_applicables is True:
```

Comment:
This is unrelated and already on couple of places, but it feels odd to compare bool variable to a bool constant to get its bool value.
```suggestion
    if return_applicables:
```